modules/running-your-dynamic-plug-in.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins.adoc
+
+:_content-type: PROCEDURE
+[id="running-your-dynamic-plugin"]
+= Running your dynamic plug-in
+You can run the plugin using a local development environment. The OpenShift console will run in a container connected to the cluster you have logged into.
+
+[id="run-plug-in-locally"]
+== Run your plug-in locally
+
+.Prerequisites
+* You must have the OpenShift CLI link:https://mirror.openshift.com/pub/[(`oc`)] installed.
```

Comment:
I'm not sure the best link for downloading `oc`, but I don't know that it's mirror.openshift.com. I think it's reasonable to assume they already have it and not have a link.